Update documentation to new structure (#1073)
Legal-and-General/canopy: projects/canopy/src/lib/docs/principles/accessibility/overview.stories.mdx

import { Meta } from '@storybook/addon-docs';
import Feedback from '../../common/feedback.md';

<Meta title="Principles/Accessibility/Overview" parameters={{ viewMode: 'docs', previewTabs: { canvas: { hidden: true } } }} />

# Overview

> "The power of the Web is in its universality.
> Access by everyone regardless of disability is an essential aspect."
>
> <cite>Tim Berners-Lee, W3C Director and inventor of the World Wide Web</cite>

## Introduction

> Before diving into explaining some of the key aspects of web accessibility here are some interesting facts to make you understand how important it is to create websites that are accessible by any user no matter their disabilities:

Reviewer: I wonder if its better to say **help** you understand, instead of **make** you understand?

> - there are around 14.6 million disable people in the UK (almost one in five of the population).

Reviewer: Should this be, "there are around 14.6 million disable**d** people in the UK..."?

- only 17% of disabled people were born with their disabilities. The majority of disabled people acquire their disability later in life.

> - for an average business, disabled people may account for up to 20% of all costumers. Many businesses fail to recognise the business opportunity that improved access presents.

Reviewer 2: ```suggestion
- for an average business, disabled people may account for up to 20% of all customers. Many businesses fail to recognise the business opportunity that improved access presents.
```

- the main reasons for disabled customers switching to a more accessible competitor include inaccessible websites.

## What is Web accessibility?

Web accessibility means that websites, tools, and technologies are designed and developed so that people with disabilities can use them.

Web accessibility isn’t just a moral requirement but also a legal one:

> [...] website owners have been obliged by law to ensure that their websites are accessible to all users, as it is illegal to treat those with disabilities less favourably.
>
> <cite>Equality Act of 2010</cite>

> In order to achieve this we have to start thinking and learning about how other users use our websites and applications; in what circumstances do they use it.

Reviewer 2: ```suggestion
In order to achieve this we have to start thinking and learning about how all users use our websites and applications, and in what circumstances they use them.
```

I don't really like 'othering' people who use assistive technologies. I know that wasn't the intent, of course :)

> ### Common challenges that we have to think about

Reviewer: I wonder if this could be Heading 2 as this is the only subheading under "What is Web accessibility"?.

Author: It's correct for this to be an h3 as it's within the context of the `What is Web accessibility` heading

> Accessibility means more than just making a website work for people with visual impairment or installing a ramp to enable wheel chair access. There are many types of impairments that can be permanent, temporary or even situational. Here are some different examples:

Reviewer 2: ```suggestion
Accessibility means more than just making a website work for people with visual impairment or installing a ramp to enable wheelchair access. There are many types of impairments that can be permanent, temporary or even situational. Here are some different examples:
```

Author: I'm not sure that sounds right 🤔 did you mean to write it in conjunction with something else?

<table>
  <thead>
    <tr>
      <th></th>
      <th>Permanent</th>
      <th>Temporary</th>
      <th>Situational</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Visual impairments</td>
      <td>
        <ul>
          <li>Blindness</li>
          <li>Colour blindness</li>
        </ul>
      </td>
      <td>
        <ul>
          <li>Low-level vision</li>
          <li>Cataract</li>
        </ul>
      </td>
      <td>Distraction</td>
    </tr>
    <tr>
      <td>Mobility impairments</td>

>       <td>Physical issuesNeurological disordersGenetic disorders</td>

Reviewer 2: These are messed up

      <td>Injury</td>

>       <td>On the phoneNew parent</td>

Reviewer 2: These are messed up

    </tr>
    <tr>
      <td>Speech impairments</td>
      <td>Non-verbal</td>
      <td>Laryngitis</td>
      <td>Heavy accent</td>
    </tr>
    <tr>
      <td>Hearing impairments</td>
      <td>
        <ul>
          <li>Low hearing levels</li>
          <li>Deafness</li>
        </ul>
      </td>
      <td>Ear infection</td>
      <td>
        <ul>
          <li>Near heavy traffic</li>
          <li>Near a sleeping person</li>
        </ul>
      </td>
    </tr>
    <tr>
      <td>Cognitive impairments</td>
      <td>
        <ul>
          <li>Intellectual disabilities</li>
          <li>Learning disabilities</li>
        </ul>
      </td>
      <td>
        <ul>
          <li>Flu</li>
          <li>Concussion</li>
        </ul>
      </td>
      <td>
        <ul>
          <li>Intoxicated</li>

>           <li>Non native speaker</li>

Reviewer 2: ```suggestion
          <li>Non-native speaker</li>
```
